src/appsignal/cli.py

```diff
@@ -135,3 +150,54 @@ def _write_file(self):
             )
 
             f.write(file_contents)
+
+
+class DemoCommand(AppsignalCLICommand):
+    def __init__(self, push_api_key=None, application=None):
+        self._push_api_key = push_api_key or os.environ.get("APPSIGNAL_PUSH_API_KEY")
+        self._name = application
```

Comment:
If we listen to `APPSIGNAL_PUSH_API_KEY`, we probably also want to listen to `APPSIGNAL_APP_NAME`.

```suggestion
        self._push_api_key = push_api_key or os.environ.get("APPSIGNAL_PUSH_API_KEY")
        self._name = application or os.environ.get("APPSIGNAL_APP_NAME")
```

The demo command in the other integrations also allows passing the environment as an argument. We might want to allow that here as well. It defaults to "development" if not given, which is probably okay -- no need to ask for it via input.

Reply:
> no need to ask for it via input.

In my opinion, none of these options need to support a prompt for this command. Keep it if it's easier, but otherwise remove it.

Reply:
Do note that we're currently not reading from the project's config file to get these values (this is the same issue that @luismiramirez just fixed for Node.js)